### DISCUSSION TITLE
PyArmor Old

### DISCUSSION THREAD

**Discussion by kittycat-ribbon** [dashingsoft/pyarmor#1950] [General]
Regarding the license. 'PyArmor Old' has disappeared from the Purchase Page. How should we proceed if we want to use the legacy version in the future?

**Comment by jondy** [comment]
Good question.
If really need Pyarmor Old license, email to pyarmor@163.com to get purchase link.
And in Pyarmor 9, it may support prior to Python 3.7 with less features like RFT/BCC etc.
